projects/plugins/boost/app/assets/src/js/lib/utils/sanitize-css.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Sanitizes CSS to prevent WAF false positives.
+ * Make sure to update the PHP desanitization function
+ * when this one is updated.
+ *
+ * @param css - The CSS to sanitize.
+ * @return The sanitized CSS.
+ */
+export default function sanitizeCSS( css: string ): string {
+	return css.replace( /xmlns/g, '__JB_XMLNS__' );
```

Comment:
I was thinking of having one central place for these, instead of having to rely on reading a comment and remembering to update the PHP counterpart.

However, I feel like it might make things more complicated, so I just kept it simple for now.

I think if we need to add more keywords, we can think about improving this.

Reply:
I had the same train of thought as you. I was thinking about a readonly DS store that serves the map to front-end from php. But, that's not something we need with only one value.